### PR TITLE
Allow buckets to be force destroyed.

### PR DESCRIPTION
**Pull request by kitsunde**
This change doesn't affect the current behavior.

### CODE REVIEW

File: terraform.tf [hunk 1]
```diff
@@ -5,6 +5,7 @@ provider "aws" {
 resource "aws_s3_bucket" "default" {
   bucket = "${var.bucket}"
   acl    = "public-read"
+  force_destroy = "${var.force_destroy}"
```
Comment (justincampbell) [inline terraform.tf]: This needs `terraform fmt`

**Review by justincampbell — APPROVED** [review]

**Comment by justincampbell** [comment]
Thanks! I've made you a collaborator, so feel free to merge these. This one could use a `terraform fmt` before merge.

**Comment by kitsunde** [comment]
Pushed `terraform fmt`, I'm not a collaborator anymore. :P

**Comment by justincampbell** [comment]
@kitsunde Thanks! It says the invite was never accepted. You should be able to see it here: https://github.com/justincampbell/terraform-aws-staticsite/invitations

**Pull request merged**